src/libslic3r/GCode/PreviewData.cpp

```diff
@@ -496,23 +486,23 @@ size_t GCodePreviewData::memory_used() const
         sizeof(shell) + sizeof(ranges);
 }
 
-const std::vector<std::string>& GCodePreviewData::ColorPrintColors()
+const std::vector<std::string>& ColorPrintColors()
```

Comment:
This is a static method which needs to be qualified. Looks like a typo, as it doesn't build because of this.

Reply:
Hmm yeah I'm not sure why I did this. I think it was building for me in VS though :thinking:

Reply:
I think I meant only to move the operators below outside of the class